README.md

```diff
@@ -46,31 +36,26 @@ also be changed.
 
 * Update the template constants (install dates and deadlines) in config/template_values.yml.
 
-## Initialize the Database
```

Comment:
we still want to include the db reset information... I had a lot of trouble getting started on this project because I didn't know how to get the environment going